models/public/yolo-v2-tiny-tf/yolo-v2-tiny-tf.md

```diff
@@ -51,8 +51,8 @@ Accuracy metrics obtained on COCO\* validation dataset for converted model.
 
 | Metric | Value |
 | ------ | ------|
-| mAP    | 27.34 |
 | [COCO\* mAP](http://cocodataset.org/#detection-eval) | 29.11 |
+| mAP    | 27.34 |
```

Comment:
in some cases you use MAP as metric name, but here mAP, it is the same metric, could you please align?

Reply:
Done